Components/wishList.css

```diff
@@ -0,0 +1,31 @@
+.wish-list-container{
+    margin-left: 6rem;
+    flex-wrap: wrap;
+    width: 70rem;
+    height: auto;
+    margin-bottom: 1rem;
+}
+.wishlist-label{
+    color: var(--color-error);
+    position: relative;
+    text-align: center;
+    left: 3rem;
+    font-size: 2rem;
+}
+.whishlist-btn{
+    margin: 2px;
+    border-radius: 10px;
+}
+.vertical-card-button > .whishlist-btn{
+    padding: 3px;
+}
+.vertical-card-button:hover > .whishlist-btn{
+    padding: 3px;
+}
```

Comment:
Properties are same for both of them. Use comma separator like 
`.vertical-card-button > .whishlist-btn, 
.vertical-card-button:hover > .whishlist-btn{
    padding: 3px;
}`

Reply:
Thanksfor poinint it out,  implemented.